### PR TITLE
Fix golint failures of pkg/kubectl/apps

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Fix golint failures of pkg/kubectl/apps

**Which issue(s) this PR fixes**:

Ref : #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```
/priority important-soon

### CODE REVIEW

File: pkg/kubectl/apps/kind_visitor.go [hunk 1]
```diff
@@ -79,11 +79,26 @@ type NoOpKindVisitor struct{}
 
 var _ KindVisitor = &NoOpKindVisitor{}
```
Comment (neolit123) [inline pkg/kubectl/apps/kind_visitor.go]: /hold
`NoOpKindVisitor{}` is technically unused.
so it seems the is no need to define it and it's methods.

`TestKindVisitor` already implements the interface for doing unit testing.

my vote would be to remove it, unless there are plans for it?
/assign @soltysh @pwittrock

Comment (SataQiu) [reply pkg/kubectl/apps/kind_visitor.go]: Thanks @neolit123 
If it is useless, I'd be happy to clean it up.

**Comment by fejta** [comment]
/approve
for hack change

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

File: pkg/kubectl/apps/kind_visitor.go (hunk 1 above)
Comment (SataQiu) [reply pkg/kubectl/apps/kind_visitor.go]: /hold cancel

**Comment by fejta** [comment]
I would recommend picking a couple people from each OWNERS file and assigning them directly. 

https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES defines aliases of people

**Comment by SataQiu** [comment]
Thanks @fejta 
/assign @soltysh @smarterclayton @liggitt

**Pull request merged**